### PR TITLE
[RDY?] Fixed martial arts belts json

### PR DESCRIPTION
**Pull request by angoddu**
Added covers, coverage, encumbrance, material_thickness, and waist flag to all the martial arts belt

### CODE REVIEW

**Comment by Zireael07** [comment]
Seriously? Coverage and encumbrance for martial arts belt? You know it's like a strip of material...

**Comment by Coolthulhu** [comment]
They all have identical stats, they should all be defined by `copy-from` from one common abstract definition.

**Comment by kevingranade** [comment]
Am I missing something, or did you set a belt to have a 20% chance of blocking any attack directed at the players torso?
If these do need coverage etc, I feel like they should be more like coverage 1 encumbrance 1.
More likely they should remain cosmetic and statless.

**Comment by angoddu** [comment]
I based all of these numbers off of the tool belt.

> You know it's like a strip of material

As someone who has been doing martial arts since I was 5 years old I know that judo belts are actually made of a very thick cotton fabric designed to not rip when used as a grab point when throwing someone. It is about 2-3 times as thick as a standard leather belt and is wrapped around the waist twice with the knot that is made to attach the belt properly can definitely get in the way

I really like having these belts in the game and I wanted them to have proper implementation. If the numbers should get changed then I can change them but they at least needed to be marked as torso, waist.

**Comment by angoddu** [comment]
> They all have identical stats, they should all be defined by copy-from from one common abstract definition.

I could do this

**Comment by Zireael07** [comment]
I know the belts are thick (a friend of mine has 1 dan in taekwondo) but they are still belts, i.e. fairly narrow strips of fabric. Even if thick fabric.

**Comment by BorkBorkGoesTheCode** [comment]
>I think the issue is I based my numbers on tool belt and not on leather belt. I will make the coverage much smaller (like 5) and encumbrance to 2

Good idea. Tool belts as present would have pouches attached, and may be wider across.

**Comment by BorkBorkGoesTheCode** [comment]
Thank you for adding the belts to a bodypart. Having them exist in hammerspace has bothered me.

File: data/json/items/armor.json [hunk 1]
```diff
@@ -5146,94 +5146,77 @@
...
-    "description": "A belt worn by martial arts practitioners, advertising a master.",
+    "name": "judo belt template",
+    "description": "This is a template for judo belts. If found in a game it is a bug.",
     "weight": 200,
     "volume": 1,
     "price": 1000,
     "to_hit": -1,
     "material": [ "cotton" ],
     "symbol": "[",
+    "color": "pink",
```
Comment (kevingranade) [inline data/json/items/armor.json]: The abstract definition shouldn't have any elements that the individual instances override, in this case color.

Comment (angoddu) [reply data/json/items/armor.json]: Ok. I thought it needed a default value. Should I remove the description as well?

**Pull request merged**